### ISSUE TITLE
 senc box 

### ISSUE THREAD

**Issue by 3052** [miquels/mp4#2]
is it supported?

**Comment by miquels** [comment]
No, `senc` is the `Sample Encryption Box` and I'm not planning to add any encryption or licensing support.

**Comment by 3052** [comment]
I wrote my own with `senc`:

https://github.com/3052/sofia
